### ISSUE TITLE
nzbToMedia.py errors with "...No module named '_sqlite3"

### ISSUE THREAD

**Issue by slash-dot** [clinton-hall/nzbToMedia#1751]
**Describe the bug**
Sabnzbd jobs from Radarr fail to post-process using nzbToMedia.py 

**Technical Specs**
1. Running on iocage jail on FreeBSD using FreeNAS
2. Using Python v 3.7
3. Using Download Client SABnbzd
4. Intended Media Management is Radarr 

**Expected behavior**
Radar:  Successfully Post Processed <<name of job>>

**Log**
Post-Processing messages shown by sabnzbd:

Changing to directory: /usr/local/share/nzbToMedia

-- Cleaning bytecode --
b'Removing __pycache__/\n'
b'Removing __pycache__/\n'

-- Cleaning folders: ['libs', 'core'] --
No folders to clean

Returning to directory:  /usr/local/share/sabnzbdplus

-- Cleanup finished --

Traceback (most recent call last):
File "/usr/local/share/nzbToMedia/nzbToMedia.py", line 730, in <module>
import core
File "/usr/local/share/nzbToMedia/core/__init__.py", line 52, in <module>
from core import logger, main_db, version_check, databases, transcoder
File "/usr/local/share/nzbToMedia/core/main_db.py", line 11, in <module>
import sqlite3
File "/usr/local/lib/python3.7/sqlite3/__init__.py", line 23, in <module>
from sqlite3.dbapi2 import *
File "/usr/local/lib/python3.7/sqlite3/dbapi2.py", line 27, in <module>
from _sqlite3 import *
ModuleNotFoundError: No module named '_sqlite3'

**Comment by clinton-hall** [comment]
Look at #1748 
It appears there is an issue with Free BSD Python3.7
Install the Python3.8 package and it should work.

**Comment by slash-dot** [comment]
Thank you for having a look and mentioning #1748.  I did look at #1748.  Do you have any idea where I might find out how to install python 3.8?  The pkg repository for FreeBSD only has version 3.7.  I searched the web and didn't see anything that explained how one can make that change inside of a freeBSD jail.  (I have python 2.7 too, but the python sym link points to 3.7).

**Comment by clinton-hall** [comment]
@silentpiranha 
Are you able to provide some details as to how you resolved this in your FreeBSD Jail?

**Comment by silentpiranha** [comment]
Needed to retrace my own steps, tried a few 😄 
(All done from inside jail CLI)

I installed the package called py38-sqlite3
Then changed symlink to new python version

`rm /usr/local/bin/python`
`ln -s /usr/local/bin/python3.8 /usr/local/bin/python`


I don't know if it is necessary, but if you can't find the pkg you need to unlock the repository to see all FreeBSD pkgs

`nano /usr/local/etc/pkg/repos/FreeBSD.conf`
switch "no" to "yes" an update pkg

**Comment by slash-dot** [comment]
It worked!  Thank you very much for your help, especially the clear directions to enable python3.8.

I tried without unlocking the repository but the package manager said it wasn't fully installing the python 3.8 package because the system is on 3.7.  So I uninstalled py38-sqlite3, edited the FreeBSD.conf to "yes" then pkg update, then repeted the process you outlined.  It worked, but also installed about 120 additional packages.
